guides/v3.2.0/models/relationships.md

```diff
@@ -190,7 +190,7 @@ export default PaymentMethod.extend({
   linkedEmail: DS.attr(),
 
   obfuscatedIdentifier: computed('linkedEmail', function () {
-    let last5 = this.get('linkedEmail').split('').reverse().slice(0, 5).reverse().join('');
+    let last5 = this.linkedEmail.split('').reverse().slice(0, 5).reverse().join('');
```

Comment:
does this work with Models? I have in the back of my mind that you still need to use `this.get()` for some things and I have a feeling that it has something to do with Ember Data but I don't know the specifics... Can anyone confirm?

Reply:
Will verify. ES5 getters can't be used for anything that implements a `Proxy` and Ember Data is using them a lot.